pkg/tag/group_builder.go

```diff
@@ -16,6 +16,14 @@ func NewGroup() *GroupBuilder {
 	return &GroupBuilder{g: &Group{}}
 }
 
+func GroupFromTag(t Tag) *Group {
```

Comment:
```suggestion
func GroupFrom(t Tag) *Group {
```

Since it is clear from the package name and type signature that it accepts tags, we can make the function name shorter

- `tag.GroupFromTag`: "tag" is repeated twice
- `tag.GroupFrom`: good